packages/@react-spectrum/s2/intl/nb-NO.json

```
@@ -2,8 +2,9 @@
   "actionbar.actions": "Handlinger",
   "actionbar.actionsAvailable": "Tilgjengelige handlinger.",
   "actionbar.clearSelection": "Tøm utvalg",
-  "actionbar.selected": "Valde element: {count}",
+  "actionbar.selected": "å{count, plural, =0 {Ingen valgt} other {# valgt}}",
```

not sure if the å is supposed to be there. just wanted to clarify

```suggestion
  "actionbar.selected": "{count, plural, =0 {Ingen valgt} other {# valgt}}",
```

---

@yihuiliao it is in the vendor translation, but I compared against other similar strings in Norwegian (against other projects, not just react-spectrum) its not in other translations, so I removed it. Thanks for catching this, I'll follow up with translator, and we can adjust again if needed, for next release, but current version should be okay for now (with  å removed)